protobuf/shared.proto

```diff
@@ -64,6 +64,11 @@ message QueryReference {
   bytes fingerprint = 3;
 }
 
+message QueryPlanReference {
+  int32 query_idx = 1;
+  int64 plan_id = 2;
```

Comment:
Should we call this `original_plan_id` here as well, for clarity? (in case we ever assign a collector-side plan fingerprint)

Reply:
Good point, let's update the name, I think it's a good change 👍